.travis.yml

```diff
@@ -30,7 +30,7 @@ env:
     - CHROME_BIN=google-chrome-stable
     - GDB_HEAP_SIZE="2G"
     - TAG=${TRAVIS_BRANCH//\//-}
-    - GRAPHDB_VERSION="8.6.0-se"
+    - GRAPHDB_VERSION="8.5.0-se"
```

Comment:
Could you possibly send Ontotext a message to ask them about the increased memory requirements of 8.6 and whether there's anything we can do about that?

Reply:
yes, wanted to do that.

Reply:
I think that we can reproduce this with the same set of GraphDB queries from #937.